renku/core/session/renkulab.py

```diff
@@ -402,14 +405,23 @@ def session_stop(self, project_name: str, session_name: Optional[str], stop_all:
         """Stops all sessions (for the given project) or a specific interactive session."""
         responses = []
         if stop_all:
-            sessions = self.session_list(project_name=project_name, config=None)
+            sessions = self.session_list(project_name=project_name)
             for session in sessions:
                 responses.append(
                     self._send_renku_request(
                         "delete", f"{self._notebooks_url()}/servers/{session.id}", headers=self._auth_header()
                     )
                 )
                 self._wait_for_session_status(session.id, "stopping")
+        elif not session_name:
+            sessions = self.session_list(project_name=project_name)
+            if len(sessions) == 1:
```

Comment:
We should print some warning if there's more than 1 session and not name/stop_all is specified. like `More than one session found, please specify a name of the session to stop or use --stop-all`. Or maybe in renku.ui.cli.session if the result is false?

Reply:
I've changed the `session_stop` method to return a status of what has been stopped. Based on that, I show the error in `core/session/session.py`.